### PR TITLE
Going through the rust dev tools contributing checklist

### PR DESCRIPTION
**Pull request by fitzgen**
Filling in some obvious gaps from https://github.com/nrc/dev-tools-team/blob/master/contribution/checklist.md

r? @emilio 

cc @rust-lang/dev-tools : would anyone like to review our CONTRIBUTING.md file in general? (and our contribution workflow?) (and probably outside of this PR)

### CODE REVIEW

**Comment by fitzgen** [comment]
Huh, I can't cc `@rust-lang/dev-tools` no matter how hard I try...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -176,6 +184,27 @@ $ cargo fmt
...
+changes should be squashed into the original commit.
+
+Unsure who to ask for review? Ask any of:
+
+* `@emilio`
+* `@fitzgen`
+
+More resources:
+
+* [Servo's Github Workflow](https://github.com/servo/servo/wiki/Github-workflow)
```
Comment (atouchet) [inline CONTRIBUTING.md]: Real minor nitpick but the H in GitHub is supposed to be capitalized.

Comment (fitzgen) [inline CONTRIBUTING.md]: Thanks!

**Comment by fitzgen** [comment]
Thanks for review!

@bors-servo r=emilio

**Pull request merged**